develop/develop-images/dockerfile_best-practices.md

```diff
@@ -198,18 +201,18 @@ COPY hello.c ./
 EOF
 ```
 
-> **Under the hood**
+> **Note**
 >
-> When building an image using a remote Git repository as build context, Docker 
+> When building an image using a remote Git repository as build context, Docker
 > performs a `git clone` of the repository on the local machine, and sends
-> those files as build context to the daemon. This feature requires `git` to be
+> those files as build context to the daemon. This feature requires Git to be
```

Comment:
This feature requires you to install Git on the host where you run the `docker build` command.